dataplane/ebpf/src/utils.rs

```diff
@@ -34,3 +37,61 @@ pub fn csum_fold_helper(mut csum: u64) -> u16 {
     }
     return !(csum as u16);
 }
+
+// Handles the termination of connection.
+// Ref: https://en.wikipedia.org/wiki/File:Tcp_state_diagram.png and
+// http://www.tcpipguide.com/free/t_TCPConnectionTermination-2.htm
+#[inline(always)]
+pub fn handle_tcp_conn_close(hdr: &TcpHdr, state: &mut TCPState) -> bool {
```

Comment:
So while I think the main _reason_ we created this function is to handle closure, in a vacuum what it actually does is something more along the lines of checking if the state needs to be transitioned, right? So similar to my above comment, do you think this naming could be helpful?

```suggestion
pub fn handle_tcp_conn_next_state(hdr: &TcpHdr, state: &mut TCPState) -> bool {
```

Or something like that? Just curious as to your thoughts :thinking:

Reply:
yeah this sounds better (sorry i suck at naming things)